packages/core/models/TransferExecModel.js

```diff
@@ -31,7 +31,7 @@ export default class TransferExecModel extends abstractModel({
     })
   }
 
-  contract (): String {
+  contract (): string {
```

Comment:
Map.get can return undefined. Flow:
```
?string
```
(the same for types below)

By the way, there is no 'contract' field in initial data above. Also 'contract' is absent in AbstractModelOld